test/extended/kubevirt/OWNERS

```diff
@@ -0,0 +1,10 @@
+reviewers:
+  - davidvossel
+  - nunnatsa
+  - orenc1
+
+approvers:
+  - bparees
```

Comment:
i don't need to be here. i'm a top level approver anyway and i have no specific reason to be an approver for this dir.

Reply:
ok, done.